CMakeLists.txt

```diff
@@ -4,4 +4,4 @@ project(spisok_2)
 set(CMAKE_CXX_STANDARD 14)
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -static-libstdc++ -static-libgcc")
 
-add_executable(spisok_2 main.cpp myqueue.h newqueue.h myqueue.cpp newqueue.cpp)
\ No newline at end of file
+add_executable(spisok_2 main.cpp myqueue.h newqueue.h myqueue.cpp newqueue.cpp Logger.h Structure.h my_exceptions.h)
```

Comment:
You can remove old queue & myqueue classes from git repo since only new classes are required. However you can save this files in your local repo if you want)